examples/Simulated_ackerman_car/simulated_ackerman.cc

```diff
@@ -0,0 +1,57 @@
+// BoB robotics includes
+#include "common/main.h"
+#include "robots/simulatedAckermanCar.h"
+#include "robots/car_display.h"
+
+// Third-party includes
+#include "third_party/units.h"
+
+// Standard C++ includes
+#include <chrono>
+
+#include <iostream>
+#include <thread>
+
+using namespace BoBRobotics;
+using namespace std::literals;
+using namespace units::literals;
+
+int
+bob_main(int, char **)
+{
+    //Robots::SimulatedTank<> robot(1.4_mps, 500_mm); // Tank agent
+    Robots::SimulatedAckermanCar<> car(1.4_mps, 500_mm);
+    Robots::CarDisplay display(10.2_m,160_mm);                     // For displaying the agent
+
+    auto mmps = 1_mps;
+    units::angle::degree_t deg = units::angle::degree_t(20);
```

Comment:
Because you've included the ``units::literals`` namespace you can write this as ``20_deg``. Just for future reference.